utils/ProgressBar.go

```diff
@@ -64,7 +65,9 @@ func NewProgressBar(label string, numberItems int) *ProgressBar {
 
 func (pb *ProgressBar) Increment() {
 	atomic.AddUint32(&pb.completedItems, 1)
+	pb.lastIncrementedMu.Lock()
 	pb.lastIncremented = time.Now() // don't care about raising sets here, it's only for a rough guess of how fast we are
+	pb.lastIncrementedMu.Unlock()
```

Comment:
Do we need this lock?

I didn't put one in because I didn't originally care about raising writes as we're only using it for estimtation of op/s

Reply:
I've taken it out. It triggers a data race if I run with `-race` without the lock though.